server/test/integration/post.spec.js

```diff
@@ -106,4 +106,81 @@ describe('Posts API', () => {
     expect(response2.status).toBe(404);
     done();
   });
+
+  it('tests the delete post method', async done => {
+    const postData = {
+      title: 'Test post',
+      body: 'This is the body for a test post',
+    };
+
+    const response = await supertest(app)
+      .post('/posts')
+      .send(postData);
+
+    const createdPost = response.body;
+    const url = '/posts/';
+
+    const response1 = await supertest(app).delete(url.concat(createdPost._id));
+
+    expect(response1.status).toBe(200);
+    done();
+  });
+
+  it('tests the deletion of already deleted post and return error message', async done => {
```

Comment:
Where is it testing that it returns an error message?

Reply:
Ah yes, nice one. That is a typo, it's meant to read "returns as error" instead.